web/src/beta/lib/reearth-ui/components/Selector/index.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import { Selector, SelectorProps } from ".";
+
+const meta: Meta<SelectorProps> = {
+  component: Selector,
+};
+
+export default meta;
+type Story = StoryObj<typeof Selector>;
+
+const LIST_ITEMS = [
+  {
+    value: "item_1",
+    label: "item_1",
+  },
+  {
+    value: "item_2",
+    label: "item_2",
+  },
+  {
+    value: "item_3",
+    label: "item_3",
+  },
+  {
+    value: "item_4",
+    label: "item_4",
+  },
+  {
+    value: "item_5",
+    label: "item_5",
+  },
+  {
+    value: "item_6",
+    label: "item_6",
+  },
+];
+
+export const Default: Story = {
+  render: () => {
+    return (
+      <div style={{ width: "300px" }}>
+        <Selector options={LIST_ITEMS} />
+      </div>
+    );
+  },
+};
+
```

Comment:
Please add one more story to show the resize feature, you only need to remove the wrapper (div) so that the selector will cover all the width and the popup should has the same width when container width changes.